backend/pick-git/src/test/java/com/woowacourse/pickgit/unit/user/application/UserServiceTest.java

```diff
@@ -584,6 +588,68 @@ void unfollowUser_SourceToTarget_Success() {
         }
     }
 
+    @DisplayName("editProfileImage 메서드는")
+    @Nested
+    class Describe_editProfileImage {
+
+        @DisplayName("새로운 프로필 이미지를 저장소에 업데이트하고 URL을 반환한다.")
+        @Test
+        void editProfileImage_WithImage_Success() throws IOException {
+            // given
+            AuthUserRequestDto authUserRequestDto = createLoginAuthUserRequestDto("testUser");
+            File imageFile = FileFactory.getTestImage1File();
+            byte[] imageSource = new FileInputStream(imageFile).readAllBytes();
+
+            // mock
+            given(userRepository.findByBasicProfile_Name("testUser"))
+                .willReturn(Optional.of(UserFactory.user(1L, "testUser")));
+            given(pickGitStorage.store(any(File.class), anyString()))
+                .willReturn(Optional.ofNullable(imageFile.getName()));
+
+            // when
+            ProfileImageEditRequestDto requestDto = ProfileImageEditRequestDto
+                .builder()
+                .image(imageSource)
+                .build();
+            ProfileImageEditResponseDto responseDto =
+                userService.editProfileImage(authUserRequestDto, requestDto);
+
+            // then
+            assertThat(responseDto.getImageUrl()).isEqualTo(imageFile.getName());
+            verify(userRepository, times(1))
+                .findByBasicProfile_Name("testUser");
+            verify(pickGitStorage, times(1))
+                .store(any(File.class), anyString());
```

Comment:
```suggestion
            assertThat(responseDto.getImageUrl()).isEqualTo(imageFile.getName());
            
            verify(userRepository, times(1))
                .findByBasicProfile_Name("testUser");
            verify(pickGitStorage, times(1))
                .store(any(File.class), anyString());
```

assertThat()이랑 verify()는 한 줄 띄워주는 게 어떤가요??